Packer.php

```diff
@@ -205,11 +205,12 @@ public function doVolumePackingWithLimitedBoxes() {
     $boxesToEvaluate = clone $this->boxes;
 
     while (!$boxesToEvaluate->isEmpty()) {
```

Comment:
So if none of the boxes fit any of the items in it when the boxes are limited then it should not throw an error rather it should just return an empty packedBoxList - for shipping nesting purposes.